spec/features/spree/sign_in_spec.rb

```diff
@@ -1,55 +0,0 @@
-# frozen_string_literal: true
```

Comment:
We shouldn't remove this spec. It used Facebook as an example but tested the generic sign-in process. We need to use another provider as an example now.

Reply:
@waiting-for-dev I now think the problem is with the `omniauth` gem itself.

The sign_in_specs fail for `omniauth-facebook`, `omniauth-github` and `omniauth-google-oauth2` when `ominauth` is >=2.0, but the sign_in_spec passes for all when `omniauth` version in <2.0 .i.e 1.9.1

Reply:
I see. Do you think it's easy to fix?